### ISSUE TITLE
Room Capacity Issue

### ISSUE THREAD

**Issue by thenorthstar** [haxball/haxball-issues#1238]
Hello @basro. I have edited the **headless-min.js** and changed room capacity identifiers from (2-30) to (0-255).

I have such a question, my rooms are visible when at 0 and 1 of capacity but they are not visible at more than 30. Frankly I couldn't give a mean that they are not visible at more if the room capacity is greater than 30. Here are the screenshots.

![1](https://user-images.githubusercontent.com/68077608/105820963-4d9bfb80-5fcb-11eb-9761-a189b7ff6de5.PNG)
Here is singleplayer room. I have tried to join and it have accepted me.

![0](https://user-images.githubusercontent.com/68077608/105820884-32c98700-5fcb-11eb-9dc4-53f477f4f286.PNG)
Here is zero capacity, I have tried to join and it have said that the room was full.

And then I have changed the identifier to a negative number tried to set a negative room capacity and it was again invisible.

To sum up, it maybe the last thing which one should think before setting a room on haxball but I remember that I had seen some rooms with at a capacity of more than 30 people and they were visible on the lobby but I can't do the same thing now. Frankly I'm curious if they knows which I don't know.

**Comment by Skarmunds** [comment]
I've played around with headless file and it doesn't seem to be possible. I do remember those larger rooms but that could be at least year ago. The room list filter goes through game JS file so I assume it can be shown on your end.

**Comment by pixiw** [comment]
Beni dcden ekleyebilir misin? Konuşmak istediğim bir konu var da, eğer eklersen çok mutlu olurum. pixiw#8210

**Comment by thenorthstar** [comment]
> Beni dcden ekleyebilir misin? Konuşmak istediğim bir konu var da, eğer eklersen çok mutlu olurum. pixiw#8210

@pixiw Buradan yeni **issue** açarak sorabilirsin.

**Comment by pixiw** [comment]
ben yeni gördüm bunu pardon

________________________________
Gönderen: thenorthstar ***@***.***>
Gönderildi: 17 Şubat 2021 Çarşamba 09:45
Kime: haxball/haxball-issues ***@***.***>
Bilgi: clara ***@***.***>; Mention ***@***.***>
Konu: Re: [haxball/haxball-issues] Room Capacity Issue (#1238)


Closed #1238<https://github.com/haxball/haxball-issues/issues/1238>.

**Comment by pixiw** [comment]
Haxballda birkaç odada gördüm, odaya giriş yapıldığında kişinin oturduğu yeri enlem ve boylam bilgisi olarak chate atıyor. Bunu nasıl yapabileceğimle alakalı yardımcı olabilir misiniz?
________________________________
Gönderen: thenorthstar ***@***.***>
Gönderildi: 17 Şubat 2021 Çarşamba 09:45
Kime: haxball/haxball-issues ***@***.***>
Bilgi: clara ***@***.***>; Mention ***@***.***>
Konu: Re: [haxball/haxball-issues] Room Capacity Issue (#1238)


Closed #1238<https://github.com/haxball/haxball-issues/issues/1238>.

**Comment by Wrathen** [comment]
@thenorthstar Welcome to the room-hosting section of the game where you have all the power! :D

After switching to HTML5 and some time later, dear Basro made the change that, rooms that have capacity over 30 and less than 0 are not shown on the room list. It used to be an unsigned byte back in Flash (0-255), and sending fake-info was much more fun back then.

**Just a bunch of fun tips;**
- You can actually dynamically change location (geo) of the room and this will make the room move in the roomList.
- You can update the name of the room dynamically. I used to show the current timer in the room name. It was cool. (when I was the only one doing it though...)
- You can make the room capacity 0 and also edit the function that handles when someone enters the room and avoid the "Room is full" error.
- You can change the amount of players currently in the room (also dynamically, just send false info to the haxball servers.)

What I also used to do is, make the maxPlayers to 30 and also send currentPlayers as a random number between 26-29 to bait people. They'd get surprised when they enter the room and see no one is there. It is actually funny to see the reactions :D

Or make a room requires password and edit the headless.js to make it accept all passwords. People who enter the room tend to think they are smart enough to beat you in their first shot trying to predict the password. Although one thing they don't know is the backstage of what is happening! :D

**Comment by thenorthstar** [comment]
OK. It seems like bad news for me. Thanks anyway!
